src/Umbraco.Cms.Api.Management/Routing/BackOfficeAreaRoutes.cs

```diff
@@ -76,7 +60,7 @@ private void MapMinimalBackOffice(IEndpointRouteBuilder endpoints)
 
         endpoints.MapControllerRoute(
             "catch-all-sections-to-client",
-            new StringBuilder(_umbracoPathSegment).Append("/{**slug}").ToString(),
+            new StringBuilder(Constants.System.UmbracoPathSegment).Append("/{**slug}").ToString(),
```

Comment:
Avoid StringBuilder here.
```suggestion
$"{Constants.System.UmbracoPathSegment)}/{{**slug}}"),
```